code-challenges.test.js

```diff
@@ -65,5 +121,45 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe('vowels', () => {
+    it("takes in a string and logs the index of the first vowel")
+    const vowelTester1 = "learn"
+    const vowelTester2 = "academy"
+    const vowelTester3 = "challenges"
+    expect(vowelTester1).toEqual(vowelTester1)
+    expect(vowelTester2).toEqual(vowelTester2)
+    expect(vowelTester3).toEqual(vowelTester3)
+
+})
+
+//Pseudocode
+// declare variable 'vowels'
+// make a  3 parameters 'vowelTester1, vowelTester2, vowelTester3
+// use .filter to only return index []
+
+const vowels = (vowelTester1, vowelTester2, vowelTester3) => {
+    return vowelTester1, vowelTester2, vowelTester3 => {
+        value.
+    }
+
+}
+
+
 
 // b) Create the function that makes the test pass.
+
+//---------------------- NOTES-------------------------
+// - had alot of problems with the problems with the challenges
+
+     //All the problems read outputed below and unsure what that means
+// > 3 | const { default: test } = require("node:test")
+// |                           ^
+// 4 | const { isTypedArray } = require("util/types")
+// 5 | const { describe, array } = require("yargs")
```

Comment:
Yes, VS code sometimes will throw these into your jest file when running jest, and they actually will break all of your tests.  If you ever see something appear at the top of your jest file that says something like `const { describe, array } = require("yargs")`, either remove them or comment them out.